File: _TheOriginalAvocado/BookStore/Models/Comment.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace BookStore.Models
{
    public class Comment
    {
        [Key]
        public int ID { get; set; }
        public string Title { get; set; }
        public string Content { get; set; }
        public int UserID { get; set; }
        public int BookID { get; set; }
        public virtual User User { get; set; }
        public virtual Book Book { get; set; }
    }
}

File: models/comment.js
var mongoose = require('mongoose');
var Schema = mongoose.Schema;

var Comment = new Schema(
    {
        Title: String,
        Content: String,
        user: String,
        book: {
            type: Schema.ObjectId,
            ref: 'Book'
        }
    }
);

module.exports = mongoose.model('Comment', Comment);
